test/test-helper.el

```diff
@@ -15,7 +17,8 @@
               indent-tabs-mode nil)
 
 ;; Load the elixir-mode under test
-(require 'elixir-mode)
+;; might be alchemist
+;; (require 'elixir-mode)
```

Comment:
Can you explain why this gets commented out here instead of detecting whether or not it should be run?